data_structures/src/chain.rs

```diff
@@ -8,6 +8,9 @@ use std::convert::AsRef;
 use std::fmt;
 use witnet_crypto::hash::{calculate_sha256, Sha256};
 
+/// Current version used when creating [`Transaction`](Transaction)s
+const TRANSACTION_VERSION: u32 = 0;
```

Comment:
I would add here:
```rust
// TODO: move TRANSACTION_VERSION into a consensus constant
```

Reply:
Then I'm removing the constant and passing it as a parameter to the factory